CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## [1.1.6] - May 5, 2022 (Unreleased)
 
 - Fixed [#62](https://github.com/SimformSolutionsPvtLtd/flutter_showcaseview/issues/62) - While using ShowCase widget, not scrolling to respective widget when its not visible.
+- Add flutter 3.0 support.
```

Comment:
There are some changes that we still need to make before we publish a version for Flutter 3.

Reply:
@BirjuVachhani  Can you please share the details on which changes are you refering ?